packages/rulesets/src/oas/index.ts

```diff
@@ -707,5 +707,23 @@ const ruleset = {
         },
       },
     },
+    'oas3_1-servers-in-webhook': {
+      message: 'Servers should not be defined in a webhook.',
+      formats: [oas3_1],
+      recommended: true,
+      given: ['$.webhooks.servers'],
+      then: {
+        function: undefined,
+      },
+    },
+    'oas3_1-callbacks-in-webhook': {
+      message: 'Callbacks should not be defined in a webhook.',
+      formats: [oas3_1],
+      recommended: true,
+      given: ['$.webhooks[*][*].callbacks'],
+      then: {
+        function: undefined,
```

Comment:
are these required?

Reply:
Are you referring to the `function: undefined` part?  That function is checking if callbacks is undefined and if it's not then it'll show the warning.